samples/drivers/watchdog/prj.conf

```diff
@@ -1,5 +1,2 @@
-CONFIG_LOG=y
```

Comment:
Are you sure this should be removed from the sample prj.conf?

Reply:
Tell me more? What types of things are you thinking of? As in the commit comment, the sample itself doesn't use logging and instead uses printk.

Reply:
But drivers might be logging useful debug info (especially in the context of a sample as these can help understand what's happening under the hood) which is why the sample enables logging and debug level for wdt drivers

See e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/watchdog/wdt_max32.c#L180

In general, even if implementing such a change would be the right thing to do, you would want to make it a different commit as it's unrelated to the addition of the WCH driver (think of something that one might want to cherry pick or revert independently)

Reply:
Figured the issue: sample asks for WDT_FLAG_RESET_SOC, not WDT_FLAG_RESET_CPU_CORE, and setup is next for ask 0 instead of whatever it is, that last one seems correct, but is reset cpu core instead of reset soc correct? Still works otherwise with the right options.

Reply:
re: logging, I've reverted the main change and added an override for ch32v003evt.

re: SOC, not sure how that regressed. Fixed.

Reply:
Ah, it regressed as I was working on two different machines and force-pushed. Fixed.